src/binding/python/RecordComponent.cpp

```diff
@@ -539,53 +539,51 @@ void load_chunk(
         }
     }
 
+    auto load_data =
```

Comment:
Let's add the same comment we add in store so we know what's going on here :)
```suggestion
    // here, we increase a reference on the user-passed data so that
    // temporary and lost-scope variables stay alive until we flush
    // note: this does not yet prevent the user, as in C++, to build
    //       a race condition by manipulating the data they passed
    auto load_data =
```